lib/reverse_sentence.rb

```diff
@@ -1,6 +1,24 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity:
+# Space complexity:
 def reverse_sentence(my_sentence)
-  raise NotImplementedError
+  return nil if my_sentence.nil?
+
+  word_array = my_sentence.split(/\.?\s+/)
```

Comment:
The problem is that you're not handling where there could be more than 1 space between things.